settings/src/main/java/org/odk/collect/settings/importing/SettingsImporter.kt

```diff
@@ -67,12 +67,8 @@ internal class SettingsImporter(
         loadDefaults(generalSettings, generalDefaults)
         loadDefaults(adminSettings, adminDefaults)
 
-        for ((key, value) in generalSettings.getAll()) {
```

Comment:
I think it was just a mistake that we implemented it in this way. If we load settings in bulk we just need to update propertyManager and reschedule tasks once.
But if I'm wrong and missing something let me know.

Reply:
Yeah, it doesn't make sense to me any more either. I think `SettingsChangeHandler` might have been introduced to fire after specific keys changed and never then reworked for "bulk" changes.